crates/orderbook/src/database/orders.rs

```diff
@@ -379,7 +379,7 @@ impl LimitOrderCounting for Postgres {
                 &Amounts {
                     sell: big_decimal_to_u256(&order_with_quote.order_sell_amount).unwrap(),
                     buy: big_decimal_to_u256(&order_with_quote.order_buy_amount).unwrap(),
-                    fee: big_decimal_to_u256(&order_with_quote.order_fee_amount).unwrap(),
+                    fee: 0.into(),
```

Comment:
The `is_order_outside_market_price` function doesn't take into account the order's fee amount anymore. Does this change make sense?